### PR TITLE
[OCPBUGS-15211]: Steps to disable autoscalers

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.10+

Issue:
[OCPBUGS-15211](https://issues.redhat.com//browse/OCPBUGS-15211)

Link to docs preview:
[Disabling autoscaling](https://61853--docspreview.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#disabling-autoscaling_applying-autoscaling)

QE review:
- [x] QE has approved this change.

Additional information:
* Tidied up a few things in the assembly while I was in here

### CODE REVIEW

**Comment by kalexand-rh** [comment]
Only versions 4.11+ are still in maintenance. I am removing labels from earlier versions from this PR.

**Comment by jeana-redhat** [comment]
refreshing old PR

File: modules/deleting-cluster-autoscaler.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  -o yaml> <cluster_autoscaler_name_backup>.yaml //<2>
+----
+<1> `<cluster_autoscaler_name>` is the name of the CR that you want to delete.
+<2> `<cluster_autoscaler_name_backup>` is the name for the backup of the CR.
+
+. Delete the `ClusterAutoscaler` CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete ClusterAutoscaler/<cluster_autoscaler_name>
```
Comment (sunzhaohua2) [inline modules/deleting-cluster-autoscaler.adoc:46]: May we can set `<cluster_autoscaler_name>` to `default` directly, as in cluster there can only be one clusterautoscaler and the name can only be default.

Comment (jeana-redhat) [reply modules/deleting-cluster-autoscaler.adoc:46]: Yes, I think that makes a lot of sense! Thanks for the suggestion :)

**Review by elmiko — COMMENTED** [review]
instructions lgtm

**Comment by jeana-redhat** [comment]
Thanks, Mike! @sunzhaohua2 let me know if my update looks alright to you - if so, I can get it published this week :nerd_face:

**Comment by sunzhaohua2** [comment]
LGTM

File: modules/deleting-cluster-autoscaler.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
++
+.Expected output
+[source,terminal]
+----
+No resources found
+----
+
+.Next steps
+
+* Disabling the cluster autoscaler by deleting the `ClusterAutoscaler` CR prevents the cluster from autoscaling, but does not delete any existing machine autoscalers on the cluster. To clean up unneeded machine autoscalers, see "Disabling a machine autoscaler".
```
Comment (bburt-rh) [inline modules/deleting-cluster-autoscaler.adoc:72]:
```suggestion
* Disabling the cluster autoscaler by deleting the `ClusterAutoscaler` CR prevents the cluster from autoscaling but does not delete any existing machine autoscalers on the cluster. To clean up unneeded machine autoscalers, see "Disabling a machine autoscaler".
```
- Tiny little nit: no comma needed before a subordinating conjunction.

File: modules/deleting-machine-autoscaler.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+[source,terminal]
+----
+$ oc get MachineAutoscaler -n openshift-machine-api
+----
++
+The disabled machine autoscaler does not appear in the list of machine autoscalers.
+
+.Next steps
+
+* If you need to reenable the machine autoscaler, use the `<machine_autoscaler_name_backup>.yaml` backup file and follow the instructions in "Deploying a machine autoscaler".
```
Comment (bburt-rh) [inline modules/deleting-machine-autoscaler.adoc:70]:
```suggestion
* If you need to re-enable the machine autoscaler, use the `<machine_autoscaler_name_backup>.yaml` backup file and follow the instructions in "Deploying a machine autoscaler".
```
The more common usage in the rest of the OCP docs is `re-enable` with a hyphen.

File: modules/deleting-cluster-autoscaler.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+* If you need to reenable the cluster autoscaler, use the `<cluster_autoscaler_name_backup>.yaml` backup file and follow the instructions in "Deploying a cluster autoscaler".
```
Comment (bburt-rh) [inline modules/deleting-cluster-autoscaler.adoc:74]:
```suggestion
* If you need to re-enable the cluster autoscaler, use the `<cluster_autoscaler_name_backup>.yaml` backup file and follow the instructions in "Deploying a cluster autoscaler".
```

The more common usage in the rest of the OCP docs is `re-enable` with a hyphen.

**Review by bburt-rh — COMMENTED** [review]
A few tiny nitpicks for you to consider, but otherwise LGTM. I really like how this PR  maintains consistency between the different (but similar) procs.

**Pull request merged**